<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveMissionWidgetClass</class>
 <widget class="QMainWindow" name="MoveMissionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoveMissionWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>执行者:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExecuterEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>人员:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>无人机:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="HumanBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="UavBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>路径点:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="MoveListWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>115.00005</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>31.0001</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>115.00007</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>31.0004</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>115.00009</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>31.0001</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>115.00001</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>31.0006</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>300</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>下发</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>当前位置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>经度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>高度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LonEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AltEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
